chainer/links/connection/lstm.py

```diff
@@ -197,6 +211,11 @@ def __call__(self, x):
             ~chainer.Variable: Outputs of updated LSTM units.
 
         """
+        if self.upward.has_uninitialized_params:
+            in_size = x.size // len(x.data)
+            self.upward._initialize_params(in_size)
```

Comment:
I did not notice that `upward` and `lateral` were documented. Since they are, I agree that we must keep them.